screens/Profile/ProfileScreenController.ts

```diff
@@ -110,16 +110,14 @@ export function useProfileScreen({ navigation }: MainRouteProps) {
     canUseBiometrics: useSelector(authService, selectCanUseBiometrics),
     useBiometrics,
 
-    UPDATE_NAME: (name: string) =>
-      settingsService.send(SettingsEvents.UPDATE_NAME(name)),
+    UPDATE_NAME: (names) =>
```

Comment:
"Profile Screen is unused" - please check and remove if not needed.

Reply:
I haven't removed Profile related code with assumption that we might leverage this structure with upcoming features.
Made changes so that it aligns with current implementation